scapy/layers/kerberos.py

```diff
@@ -251,7 +251,7 @@ class KRB_KDC_REQ_BODY(ASN1_Packet):
                          explicit_tag=0xa3),
         ),
         ASN1F_optional(
-            KerberosTime("from", GeneralizedTime(), explicit_tag=0xa4)
+            KerberosTime("From", GeneralizedTime(), explicit_tag=0xa4)
```

Comment:
This is a bad idea to switch to `From` as other keywords start with a lower case.